app/views/pages/medicaid/amounts_current_job.erb

```diff
@@ -0,0 +1,29 @@
+<div class="step-header">
+  <a href="/pages/medicaid/amounts_overview" class="step-header__back-link">
+    <i class="button__icon icon-arrow_back"></i>
+  </a>
+  <h4 class="step-header__title">
+    Income & Expense Amounts
+  </h4>
+</div>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Tell us more about the specifics of your current job.</div>
```

Comment:
what if someone has multiple jobs?

Reply:
@jessieay great question. I was wondering the same thing. @alanjosephwilliams how are we addressing multiple jobs? Adding to Trello...